docs/charon/charon_cli_reference.md

```diff
@@ -100,16 +103,20 @@ Usage:
 Flags:
       --clean                          Delete the cluster directory before generating it.
       --cluster-dir string             The target folder to create the cluster in. (default ".charon/cluster")
+      --definition-file string         Optional path to a cluster definition file or an HTTP URL. This overrides all other configuration flags.
       --fee-recipient-address string   Optional Ethereum address of the fee recipient
   -h, --help                           Help for cluster
-      --name string                    Optional cosmetic cluster name
+      --insecure-keys                  Generates insecure keystore files. This should never be used. It is not supported on mainnet.
+      --keymanager-addresses strings   Comma separated list of keymanager URLs to push validator key shares to. Note that multiple addresses are required, one for each node in the cluster, with node0's keyshares being pushed to the first address, node1's keyshares to the second, and so on.
```

Comment:
Did we go through with the implementation approach or did we just go with a single keymanager for local? What happens if a random one of these requests fails?